draft-ietf-quic-http.md

```diff
@@ -1408,7 +1414,7 @@ SETTINGS Frame {
 ~~~~~~~~~~~~~~~
 {: #fig-ext-settings title="SETTINGS Frame"}
 
-An implementation MUST ignore the contents for any SETTINGS identifier it does
+An implementation MUST ignore the Value for any SETTINGS identifier it does
 not understand.
```

Comment:
This sentence reads weird with or without the suggestion, identifier and value are independent parts of a Setting. The text before the drawing uses "parameter" a bit. Perhaps it is clearer to say something like 

"An implementation MUST ignore parameters with an unknown identifier"?